website/components/remote-plugin-docs/utils/resolve-nav-data.js

```diff
@@ -164,13 +165,14 @@ async function resolvePluginEntryDocs(pluginConfigEntry, currentPath) {
     const sourceUrl = `https://github.com/${repo}/blob/${sourceBranch}/${filePath}`
     // determine pluginTier
     const pluginOwner = repo.split('/')[0]
-    const pluginTier = pluginOwner === 'hashicorp' ? 'official' : 'community'
+    const parsedPluginTier =
+      pluginTier || (pluginOwner === 'hashicorp' ? 'official' : 'community')
```

Comment:
We use the provided `pluginTier` property if it's available, or fall back to deriving the plugin tier from the plugin repo owner.